File: C/Assignments/Assignment 4/question_3.c
//Q3. Print perfect numbers in the given range 1 to n.

#include<stdio.h>
void main(){

    int j, n, sum;

    printf("Enter range: ");
    scanf("%d", &n);

    for(int i=1; i<=n; i++){
        sum=0;
        j=1;
        while(j<i){
            if(i%j==0)
                sum += j;
                j++;
        }

        if(sum==i)
            printf("%d is perfect number\n", i);
        
    }
}

File: CtoJava/src/Assignment_3/question_3.java
package Assignment_3;

import java.util.Scanner;
public class question_3 {
	/*Q3 Sum of numbers in given range.
	Find sum of numbers from start to end.
	Input: start = 1, end = 5
	Output: 15*/

	
	public static void main(String args[]){
	    int start, end, sum=0;

	    System.out.printf("Enter start and end: ");
	    
	    Scanner sc=new Scanner(System.in);
	    start=sc.nextInt();
	    end=sc.nextInt();

	    while(start<=end){
	        sum+=start;
	        start++;
	    }
	    System.out.printf("Sum of numbers in range: %d", sum);
	    
	    sc.close();
	}
}
